### ISSUE TITLE
[Suggestion] Frontend Developer(add new videos about internet)

### ISSUE THREAD

**Issue by Heisen19** [kamranahmedse/developer-roadmap#3267]
Hey, what's up?

I've some contents to add to roadmap.

I found a great channel on Youtube where it's talk about network and internet.

The channel is https://www.youtube.com/@PowerCertAnimatedVideos.

The videos that I think would be nice to roadmap are:

About servers -> https://www.youtube.com/watch?v=UjCDWCeHCzY

About DNS -> https://www.youtube.com/watch?v=mpQZVYPuDGU

About IPv4 and IPv6 ->https://www.youtube.com/watch?v=ThdO9beHhpA

The difference between Modem and router -> https://www.youtube.com/watch?v=Mad4kQ5835Y

Basically, these are the contents.

Thanks for help me with your roadmap!
